src/components/ButtonGroupPicker/index.d.ts

```diff
@@ -0,0 +1,17 @@
+import { BaseProps } from '../types';
+import { ChangeEvent, ReactNode, ComponentType } from 'react';
+
+export interface ButtonGroupPickerProps extends BaseProps {
+    label: string | ReactNode;
+    value: string | string[];
+    onChange: (event: ChangeEvent<HTMLElement>) => void;
```

Comment:
here instead of an event we should send the value which is:
```type Value = string[] | string;```
then you can:
```
value?: Value;
onChange?: (value: Value) => void;
```